fmn/fasjson_client.py

```diff
@@ -0,0 +1,55 @@
+import requests
+import requests.exceptions
+from requests.compat import urlencode, urljoin
+from requests_gssapi import HTTPSPNEGOAuth
+
+
+class Client(object):
+    """
+    A fasjson client to make very specific requests to fasjson.
+    Necessary because the official fasjson-client library does not support
+    python3.
+    """
+    def __init__(self, url, principal=None):
+        self.url = url
+        self.principal = principal
+
+        gssapi_auth = HTTPSPNEGOAuth(
+            opportunistic_auth=True, mutual_authentication="OPTIONAL"
```

Comment:
Looks like `mutual_authentication="OPTIONAL"` [is discouraged](https://github.com/pythongssapi/requests-gssapi#optional) by requests-gssapi. It should probably be disabled, if I understand the docs well.